README.md

```diff
@@ -20,25 +20,25 @@ Support...
+| Threading support                          | :heavy_check_mark:Ex  | :heavy_check_mark:Ex  | :heavy_check_mark:               |                    | :x:                             |                              | :heavy_minus_sign:?        | :x:                             | :x:?                | :heavy_minus_sign:? | Threading support                          |
 | World Events/Messages                      | :heavy_check_mark:    | :heavy_check_mark:    | :heavy_check_mark:               |                    | :heavy_check_mark:              |                              |                            | :x:?                            | :heavy_check_mark:  | :heavy_check_mark:  | World Events/Messages                      |
 | One-Frame Components                       | :heavy_check_mark:    | :heavy_check_mark:    |                                  |                    | :heavy_check_mark:EX            |                              |                            |                                 |                     |                     | One-Frame Components                       |
 | Dependency Injection<br/>Support           | :heavy_check_mark:    | :heavy_check_mark:    | :x:                              |                    | :x:                             | :x:                          | :x:                        | :x:                             | :x:                 |                     | Dependency Injection<br/>Support           |
 | Shared Resource                            | :heavy_check_mark:    | :heavy_check_mark:    | :heavy_check_mark:               |                    | :heavy_check_mark:              |                              |                            |                                 |                     | :heavy_check_mark:  | Shared Resource                            |
 | Thread-safe                                | :x:                   | :x:                   | :heavy_minus_sign:               | :x:                | :x:                             |                              | :x:?                       | :x:                             | :x:                 | :x:?                | Thread-safe                                |
-| Reactive System                            | :heavy_check_mark:    | :heavy_check_mark:    | :heavy_check_mark:               |                    | :x:?                            | :heavy_check_mark:           |                            | :heavy_check_mark:              |                     | :heavy_check_mark:  | Reactive System                            |
+| Reactive System                            | :x:                   | :heavy_check_mark:    | :heavy_check_mark:               |                    | :x:?                            | :heavy_check_mark:           |                            | :heavy_check_mark:              |                     | :heavy_check_mark:  | Reactive System                            |

```

Comment:
I'd define this as a reactive feature.
https://github.com/Leopotam/ecs#i-want-to-add-some-reactive-behaviour-on-filter-items-change-how-i-can-do-it

Reply:
Well, its not reactive feature, but events on filter changes. Then you can name world events as reactive feature too, but users of `entitas` will not agree with you. :)